src/circuits/utils.stanza

```diff
@@ -0,0 +1,123 @@
+#use-added-syntax(jitx)
+defpackage jsl/circuits/utils:
+  import core
+  import jitx
+  import jitx/commands
+
+  import jsl/si/signal-ends
+  import jsl/design/introspection
+
+doc: \<DOC>
+Create short-trace relationships using `signal-ends`
+
+This function is generator and is expected to run in
+a `pcb-module` context.
+
+This function uses the `find-singal-end` function to
```

Comment:
Typo in `This function uses the `find-singal-end` function to`

Change to `This function uses the `find-signal-end` function to`